### ISSUE TITLE
how to preprocess the original IXI dataset  like your format?

### ISSUE THREAD

**Issue by sunshinelike** [icon-lab/SynDiff#13]
how to preprocess the original IXI dataset on line  like your format?

**Comment by onat-dalmaz** [comment]
Thank you for your comment! First of all, sequentially select the first 40 subjects from the dataset. Since multi-contrast MRI volumes were unregistered as publicly shared in IXI, T2- and PD-weighted volumes were registered onto T1-weighted volumes in IXI Registrations were implemented in FSL via affine transformation and mutual information loss. In each subject, each imaging volume was separately normalized to a mean intensity of 1. The maximum voxel intensity across subjects was then normalized to 1 to ensure an intensity range of [0,1]. We hope this information helps clarify the preprocessing steps taken for the dataset used in our study.

**Comment by onat-dalmaz** [comment]
No we only used 40 subjects, with a (25,5,10) split for train, validation, and test sets. For validation and test sets, to be able to accurately calculate performance metrics, we have registered T2 volumes to T1 volumes. Since the method is unsupervised, there is no registration requirement for the training set.

**Comment by sunshinelike** [comment]
Thank you very much for your helpful and quick response. OK, I see.
Best wishes.

**Comment by Luke-Huang-Github** [comment]
@onat-dalmaz Hi, thank you for your contribution. I have some questions about the details of the dataset.
(1) In IXI dataset, First of all,  sequentially select the first 40 subjects from the dataset,with a (25,5,10) split for train, validation, and test sets.  In paper "each subject 100 axial cross-sections with brain tissue were selected". Do you mean the data_train_contrast1.mat shape is (2500, width, height) -> (#images, width, height)? #images = 25*100?
